### DISCUSSION TITLE
Slide only one item

### DISCUSSION THREAD

**Discussion by claudiu-zbucea** [microsoft-search/pnp-modern-search#3755] [General]
Hello
I have a slide with custom template that has only one full width item shown at a time. 
I use: {{#>slider items=(JSONstringify @root.items 0) options=(JSONstringify @root.sliderOptions)}} to make a slide.
But when I push arrow it jumps at forth item from results. Then at next push it jumps at seventh item from results an so on.
To make template full width for an item I use in style:  
        .template_root div {
            width: 100.01%;
        }

I want to go to next item when I slide. Can you tell me if is possible?

**Version used**
Ex: 3.4.1

**Desktop (please complete the following information):**
 - Chrome

**Additional context**
Ths is entire templates code used (except css):

<div class="template_root">
        {{#if @root.hasPrimaryOrSecondaryResults}}
        {{#>slider items=(JSONstringify @root.items 0) options=(JSONstringify @root.sliderOptions)}}
        <div class="slide" style="max-height: 800px;width: 100%;margin: 0px;">
            <div class="elementMainContainerSearch">
                <div class="mainNews">
                    <div class="imgMainNews">
                        <img src="\{{getUrlField item.ONEnetTeaserImageOWSURLH 'URL'}}" class="imgMainImgNews"
                            alt="\{{getUrlField item.ONEnetTeaserImageOWSURLH 'Title'}}">
                    </div>
                    <div class="textMainNews">
                        <div class="antetMainNews">
                            Created on \{{getDate item.Created 'DD.MM.YYYY'}}
                        </div>
                        <div class="titleMainNews">
                            \{{item.Title}}
                        </div>
                        <div class="containMainNews">
                            \{{item.ONEnetTeaserOWSMTXT}}
                        </div>
                        <div class="linkMainNews">
                            <a href="#" class="linkRefNews">READ MORE &#x2794</a>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        {{/slider}}
        {{else}}
        {{#unless showBlank}}
        <div class="template_noResults">{{@root.strings.NoResultMessage}}</div>
        {{/unless}}
        {{/if}}
    </div>
</content>

<content id="placeholder">

    <style>
        .flickity-page-dots {
            bottom: 5px;
        }
    </style>

    <div class="placeholder_root">
        {{#>slider items="[{},{},{},{},{}]" options=(JSONstringify @root.sliderOptions)}}
        <div class="slide" style="max-height: 800px;width: 100%;margin: 0px;">
            <pnp-document-card-shimmers data-is-compact="\{{@root.isCompact}}"></pnp-document-card-shimmers>
        </div>
        {{/slider}}
    </div>

**Comment by kasperbolarsen** [comment]
Version 3.* hasn't been supported for quite awhile.  As time is a limiting resource we only support the current version which is 4.11.1

**Comment by kasperbolarsen** [comment]
In 4.11.1 we have this option that might solve your issue
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/872d8167-0157-43ee-8696-7185715fd4cd)

**Comment by claudiu-zbucea** [comment]
As you said in ver. 4 it works well, after I put parameter "Number of elements to group..." to value 1. Even I use after a custom template that include pnp-slider tag.
